pinot-common/src/main/java/com/linkedin/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -52,32 +56,21 @@
   private SelectionResults _selectionResults;
   private List<AggregationResult> _aggregationResults;
 
-  private Map<String, String> _traceInfo;
-  private List<QueryProcessingException> _processingExceptions;
-  private List<String> _segmentStatistics;
-
-  public static final BrokerResponseNative EMPTY_RESULT;
-  public static final BrokerResponseNative NO_TABLE_RESULT;
-
-  static {
-    EMPTY_RESULT = new BrokerResponseNative();
-    EMPTY_RESULT.setTimeUsedMs(0);
+  private Map<String, String> _traceInfo = new HashMap<>();
+  private List<QueryProcessingException> _processingExceptions = new ArrayList<>();
```

Comment:
By doing this, we can ignore null handling and simplify the method adding things into these objects. Also, we don't have nulls in the broker response. (This object is the response we get from query console)